src/services/default_docs_service.py

```
@@ -0,0 +1,694 @@
+            owner_name=anonymous_user.name,
+            owner_email=anonymous_user.email,
+            file_size=os.path.getsize(temp_file_path),
+            connector_type="openrag_docs",
+            is_sample_data=True,
+        )
+    finally:
+        try:
+            os.unlink(temp_file_path)
+        except FileNotFoundError:
+            pass
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)